docs/examples/server.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
GH doesn't regularly spam admins, but on an inconsistent basis, security consulting type folks will go trawling through repositories looking for matches of that regex (I have to admit I type it in sometimes just to make myself sad; many usages are not so harmless).

The twisted examples that ship .pem files with the code are definitely the bad example I learned from here.